app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -85,7 +85,7 @@ function initializeCommentDropdown() {
     }
   }
 
-  function initializeCommentEvents(commentsContainer) {
+  function initializeDropDownClick(commentsContainer) {
```

Comment:
Renamed this as it also pertains to the post share button.

Reply:
I think we should probably rename the arg too in this case. Also maybe the file name itself should change? I wouldn't expect the post share button to be initialized within `initializeCommentDropdown.js` (probably why we missed this in the first place!) 🤔 

```suggestion
  function initializeDropDownClick(dropdown) {
```